lib/modules/platform/gitlab/index.ts

```diff
@@ -548,36 +550,14 @@ export async function getBranchStatus(
 
 // Pull Request
 
-async function fetchPrList(): Promise<Pr[]> {
-  const searchParams = {
-    per_page: '100',
-  } as any;
-  // istanbul ignore if
-  if (!config.ignorePrAuthor) {
-    searchParams.scope = 'created_by_me';
-  }
-  const query = getQueryString(searchParams);
-  const urlString = `projects/${config.repository}/merge_requests?${query}`;
-  try {
-    const res = await gitlabApi.getJsonUnchecked<GitLabMergeRequest[]>(
-      urlString,
-      {
-        paginate: true,
-      },
-    );
-    return res.body.map((pr) => prInfo(pr));
-  } catch (err) /* istanbul ignore next */ {
-    logger.debug({ err }, 'Error fetching PR list');
-    if (err.statusCode === 403) {
-      throw new Error(PLATFORM_AUTHENTICATION_ERROR);
-    }
-    throw err;
-  }
-}
-
 export async function getPrList(): Promise<Pr[]> {
   if (!config.prList) {
-    config.prList = await fetchPrList();
+    config.prList = await GitlabPrCache.getPrs(
```

Comment:
wonder if we still need `config.prList` 🤔 we don't have it on gitea.

Reply:
We don't

Reply:
let's remove it to keep things simple